index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="first-sector">
+        <div class="header-comtainer">
+            <div class="header">
+                <div class="my-logo">My Logo</div>
+                <nav class="nav-bar">
+                    <div class="menu-buttom"><a href="#"><img src="images/humburger-btn.svg" alt="menu-buttom"></a></div>
+                </nav>
+            </div>
+        </div>
+        <div class="content">
+            <section class="main">
+            <h2 class="my-name">
+                I'm Carlos <br> Glad to see you!
+            </h2>
```

Comment:
- [ ] Since this is the main title of your portfolio, I think it would be best to use an `<h1>` tag instead, this is because search engines like google look for this tag and index the page based on it. 🤓 🛠️ 

```suggestion
            <h1 class="my-name">
                I'm Carlos <br> Glad to see you!
            </h1>
```